moveos/moveos-stdlib/moveos-stdlib/sources/event.move

```diff
@@ -47,7 +47,7 @@ module moveos_std::event {
     /// Borrow a mut event handle from the object storage
     fun borrow_event_handle_mut<T>(ctx: &mut Context): &mut EventHandle {
         let event_handle_id = derive_event_handle_id<T>();
```

Comment:
`event_handle_id` can also be refactored to use singleton_object directly

Reply:
I plan to remove the EventHandle from state. #1056